application/templates/deployment.yaml

```diff
@@ -75,7 +75,10 @@ spec:
       {{- end }}
       {{- if .Values.deployment.initContainers }}
       initContainers:
-{{ include "application.tplvalues.render" ( dict "value" .Values.deployment.initContainers "context" $ ) | indent 6 }}
+  {{- range $key, $value := .Values.deployment.initContainers  }}
+        - name: {{ $key }}
```

Comment:
remove 2 spaces from here as well just to be consistent

Reply:
Done.